ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml

```diff
@@ -44,6 +44,10 @@ images:
   dockerfile_path: build/wait-for-vmi-status/Dockerfile
   from: openshift_release_golang-1.15
   to: tekton-task-wait-for-vmi-status
+- context_dir: modules/modify-vm-template/
+  dockerfile_path: build/modify-vm-template/Dockerfile
+  from: base_fedora
```

Comment:
base_fedora?

Reply:
I am trying to get rid of golang images as we discussed.

Reply:
I know, but we do not use fedora there, can't we rather use ubi?

Reply:
sure, fixed

Reply:
I would rather put the actual image that we use (`ubi-minimal`), even though now it does not seem to make a difference. Not sure why though ...

Reply:
fixed